packages/editor-ui/src/components/executions/workflow/WorkflowExecutionAnnotationPanel.vue

```diff
@@ -0,0 +1,385 @@
+		},
+	},
+});
+</script>
+
+<template>
+	<div
+		ref="container"
+		:class="['execution-annotation-panel', $style.container]"
+		data-test-id="execution-annotation-panel"
+	>
+		<div :class="$style.section">
+			<div :class="$style.vote">
+				<div>{{ $locale.baseText('generic.rating') }}</div>
+				<VoteButtons :vote="vote" @vote-click="onVoteClick" />
+			</div>
+			<span class="tags" data-test-id="annotation-tags-container">
+				<AnnotationTagsDropdown
+					v-if="isTagsEditEnabled"
+					v-model="appliedTagIds"
+					ref="dropdown"
+					:create-enabled="true"
+					:event-bus="tagsEventBus"
+					:placeholder="$locale.baseText('executionAnnotationView.chooseOrCreateATag')"
+					class="tags-edit"
+					data-test-id="workflow-tags-dropdown"
+					@blur="onTagsBlur"
+					@esc="onTagsEditEsc"
+				/>
+				<div v-else-if="tagIds.length === 0">
+					<span
+						class="add-tag add-tag-standalone clickable"
+						data-test-id="new-tag-link"
+						@click="onTagsEditEnable"
+					>
+						+ {{ $locale.baseText('executionAnnotationView.addTag') }}
+					</span>
+				</div>
+
+				<span
+					v-else
+					class="tags-container"
+					data-test-id="execution-annotation-tags"
+					@click="onTagsEditEnable"
+				>
+					<span v-for="tag in tags" :key="tag.id" class="clickable">
+						<el-tag :title="tag.name" type="info" size="small" :disable-transitions="true">
+							{{ tag.name }}
+						</el-tag>
+					</span>
+					<span class="add-tag-wrapper">
+						<n8n-button
+							class="add-tag"
+							:label="`+ ` + $locale.baseText('executionAnnotationView.addTag')"
+							type="secondary"
+							size="mini"
+							:outline="false"
+							:text="true"
+							@click="onTagsEditEnable"
+						/>
+					</span>
+				</span>
+			</span>
+		</div>
+		<div :class="$style.section">
+			<div :class="$style.heading">
+				<n8n-heading tag="h3" size="small" color="text-dark">
+					{{ $locale.baseText('generic.annotationData') }}
+				</n8n-heading>
+			</div>
+			<div
+				v-if="activeExecution?.customData && Object.keys(activeExecution?.customData).length > 0"
+				:class="$style.metadata"
+			>
+				<div
+					v-for="attr in Object.keys(activeExecution?.customData)"
+					v-bind:key="attr"
+					:class="$style.customDataEntry"
+				>
+					<n8n-text :class="$style.key" size="small" color="text-base">
+						{{ attr }}
+					</n8n-text>
+					<n8n-text :class="$style.value" size="small" color="text-base">
+						{{ activeExecution?.customData[attr] }}
+					</n8n-text>
+				</div>
+			</div>
+			<div v-else :class="$style.noResultsContainer" data-test-id="execution-list-empty">
+				<n8n-text color="text-base" size="small" align="center">
+					<span v-html="$locale.baseText('executionAnnotationView.data.notFound')" />
+				</n8n-text>
+			</div>
+		</div>
+	</div>
+</template>
+
+<style module lang="scss">
+.container {
+	z-index: 1;
+	position: absolute;
+	bottom: 0;
+	right: var(--spacing-xl);
+	transform: translate(0, 100%);
```

Comment:
Why are we positioning it using transform? 🤔 Wouldn't this be easier?
```cs
.container {
	z-index: 1;
	position: absolute;
	right: var(--spacing-xl);
	top: var(--spacing-3xl);
	max-height: calc(100vh - 250px);
	width: 250px;
```

Reply:
The parent container's height is not fixed, and may grow because of text wrapping e.g. when viewport is extremely narrow or enlarged font sizes. It seems it's safer to rely on parent's bottom coordinates.